reactive/calico.py

```diff
@@ -231,3 +232,12 @@ def deploy_network_policy_controller(etcd, cni):
 @when_any('cni.is-master', 'calico.npc.deployed')
 def ready():
     status_set('active', 'Calico is active')
+
+
+def arch():
```

Comment:
We should really consider adding this as a charmhelper.  We have this `arch()` function in a few places now (eg, flannel).